lua/kickstart/plugins/oil.lua

```diff
@@ -0,0 +1,27 @@
+return {
+  { -- Oil filesystem editor and explorer
+    'stevearc/oil.nvim',
+    dependencies = { 'nvim-tree/nvim-web-devicons' },
```

Comment:
Can you maybe make it so it respects the web-devicons config from init.lua, since it's optional?

Reply:
Great suggestion! Yes, updating accordingly.